protobuf/build.gradle

```diff
@@ -30,5 +30,5 @@ dependencies {
 }
 
 tasks.named("javadoc").configure {
-    options.links 'https://developers.google.com/protocol-buffers/docs/reference/java/'
+    options.links 'https://protobuf.dev/reference/java/api-docs/'
```

Comment:
I'm getting `Not found`

Reply:
I believe that is a prefix added to outgoing links so it would not by itself point to anything. This change already fixed the build on master so it should be correct.